### ISSUE TITLE
Error while compiling with TMC2209 UART mode

### ISSUE THREAD

**Issue by Marcussy-lw** [makerbase-mks/MKS-TinyBee#31]
![image](https://user-images.githubusercontent.com/103623250/167797511-869f36a5-f616-4242-8a28-aff704433529.png)

I get this error while compiling if i set TMC2209s up in UART mode:

![image](https://user-images.githubusercontent.com/103623250/167798222-b26b0e9c-0504-4e97-8574-1d287e85f0b0.png)

Compilation only works if i set it up in TMC2209_STANDALONE mode

**Comment by flishy2532** [comment]
use standalone

**Comment by grafalex82** [comment]
Is there a chance to enable UART support for TMC2209 on this board?

**Comment by flishy2532** [comment]
no am use tmc2209 stand alone it work

รับ Outlook สำหรับ Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by jirka-malvin** [comment]
yes UART is possible, but with complications and with no LCD. Already solved in #6

**Comment by grafalex82** [comment]
ok, thanks.

**Comment by Rmdhn1** [comment]
> yes UART is possible, but with complications and with no LCD. Already solved in #6

AFAIK you can still use LCD and not many complications, just use Hardware UART.
